arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,776 @@
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DatabaseAccountsListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatabaseAccount"
+          },
+          "description": "List of database account and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the database accounts and their properties."
+    },
+    "FailoverPolicies": {
+      "type": "array",
+      "description": "List of failover policies.",
+      "items": {
+        "$ref": "#/definitions/FailoverPolicy"
+      }
+    },
+    "FailoverPolicy": {
+      "type": "object",
+      "description": "The failover policy for a given region of a database account.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The unique identifier of the region in which the database account replicates to. Example: &lt;accountName&gt;-&lt;locationName&gt;."
+        },
+        "locationName": {
+          "type": "string",
+          "description": "The name of the region in which the database account exists."
+        },
+        "failoverPriority": {
+          "type": "integer",
+          "minimum": 0,
+          "format": "int32",
+          "description": "The failover priority of the region. A failover priority of 0 indicates a write region. The maximum value for a failover priority = (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists."
+        }
+      }
+    },
+    "Location": {
+      "description": "A region in which the Azure DocumentDB database account is deployed.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The unique identifier of the region within the database account. Example: &lt;accountName&gt;-&lt;locationName&gt;."
+        },
+        "locationName": {
+          "type": "string",
+          "description": "The name of the region."
+        },
+        "documentEndpoint": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The connection endpoint for the specific region. Example: https://&lt;accountName&gt;-&lt;locationName&gt;.documents.azure.com:443/"
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The status of the database account at the time the operation was called.",
+          "enum": [
+            "Creating",
+            "Succeeded",
+            "Updating",
+            "Deleting",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
```

Comment:
Does it make sense to provide an enum for a readOnly property? And would you really want to make this modelAsString: false? What if you decide to add a new value called "Cancelling"? Then that would be a breaking change. 
I would recommend removing the enum Array and the x-ms-enum extension.
However, if you wish to keep the enum array then make sure to make `"x-ms-enum".modelAsString: true`